web/src/components/watch.tsx

```diff
@@ -93,7 +93,7 @@ export default function Watch(props: { name: string }) {
 	return (
 		<>
 			<Fail error={error()} />
-			<div class="relative aspect-video w-full">
+			<div class="relative aspect-video w-full" id="video">
```

Comment:
I noticed the video gets stretched in fullscreen when using Chrome responsive tools. I guess it's probably because the canvas element isn't being re-sized correctly. 

<img width="819" alt="Screenshot 2024-12-13 at 8 18 09 PM" src="https://github.com/user-attachments/assets/848cd012-a072-44d8-93ce-71bed9025679" />

Reply:
yes it doesn't preserve aspect ratio @christriants

Reply:
Done ✅ Setting the _[object-contain](https://github.com/englishm/moq-js/pull/20/files#diff-781d3f09acafc49b0471cc6272eff2014705c33c79c13b5723210a7305a26436R103)_ CSS property on the canvas did the trick

![image](https://github.com/user-attachments/assets/86a889b7-b8dc-4fb8-a582-9da60e6f8f5a)

Reply:
Looks awesome!